Makefile

```
@@ -64,3 +64,13 @@ test: build/xdebug-filter.php
 docs:
 	cd docs && sphinx-build -W -b html -d _build/doctrees . _build/html
 .PHONY: docs
+
+YARN=@docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn
```

```suggestion
YARN ?= @docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn
```
This will allow overriding the variable

---

Ahhh okay nicee didn't know this! 👍